File: tarea semana 3/circulo.py
class circulo:
    def __init__(self,radio):
        self.radio=radio
    def calcularArea(self):
        pi=3.14
        x=(self.radio**2)*pi
        print("El area es ", x)
    def calcularPeri(self):
        pi=3.14
        x=2*pi*self.radio
        print("el perimetro es: ", x)
    def cambiar_radio(self,_radio):
        newradio=_radio
        self.radio = newradio
l=1
c1= circulo(2)
while l>0:
    v=str(input("Desea cambiar radio y(si) o n(no) "))
    if v=="y":
        c1.cambiar_radio(int(input("Ingrese Radio ")))
    c1.calcularArea()
    c1.calcularPeri()
    t=str(input("ingrese y para continuar o n para salir : "))
    if t=="n":
        l=0

File: tarea semana 3/circulo.cpp
#include <iostream>
using namespace std;
class circulo{
    public:
        int radio;
    public:
        circulo(int);
        void CalcularArea();
        void CalcularPerimetro();
        void CambiarRadio();
};
circulo::circulo(int  _radio){
    radio = _radio;
}
void circulo::CalcularArea(){
    float pi= 3.14;
    float x= (radio*radio)*pi;
    cout<<"el Area es ";
    cout<<x<<endl;
}
void circulo::CalcularPerimetro(){
    float pi= 3.14;
    float x= 2*pi*radio;
    cout<<"el Perimetro es ";
    cout<<x<<endl;
}
void circulo::CambiarRadio(){
    string f;
    cout<<"Desea cambiar el radio y(si) o n(no) ",cin>>f;
    int newradio;
    if(f=="y"){
        cout<<"Ingrese valor del Radio ", cin>> newradio ;
        radio = newradio;
    }
}
int main()
    {
        circulo c1= circulo(3);
        string d;
        cout<<"Desea continuar y(si) o n(no) ", cin>> d;
        while (d=="y")
        {
            c1.CambiarRadio();
            c1.CalcularArea();
            c1.CalcularPerimetro();
            cout<<"Desea continuar y(si) o n(no)", cin>>d;
        }
    }